group_vars/recap/staging.yml

```diff
@@ -0,0 +1,13 @@
+---
+recap_base_url: 'recap-staging.princeton.edu'
+
+drupal_db_user: 'recap-staging'
+drupal_db_password: "{{ recap_db_password }}"
+drupal_db_name: 'recap-staging'
+drupal_db_port: '3306'
+drupal_docroot: '/var/www/recap_cap'
+apache_app_path: '{{ drupal_docroot }}/current'
+
+db_host: 'mysql-db-staging1.princeton.edu'
+
+drupal_major_version: 7
```

Comment:
@kayiwa I think this should be drupal 9 here.  I realize that it is wrong already, but would you mind updating it.  I just do not want running the playbook to break something.